### PR TITLE
Developing uncontrast

### PR DESCRIPTION
**Pull request by oryrosin**

### CODE REVIEW

File: src/contrast/pages/ContactPage/ContactUs.jsx [hunk 1]
```diff
@@ -54,6 +54,7 @@ export default function ContactUs() {
       <div className="border-b border-black w-full"></div>
 
       <Formik
+        enableReinitialize={true}
```
Comment (ChenPeleg) [inline src/contrast/pages/ContactPage/ContactUs.jsx:57]: Good I've also learned something new!

File: src/contrast/pages/UploadNewPaper/ExperimentsSection/ExperimentForm.jsx [hunk 2]
```diff
@@ -138,6 +141,13 @@ export default function ExperimentForm({
       label: rawTextToShow(type),
     }));
   const shouldCheckAllClassificationsFilled = false;
+
+  if (experimentData) {
```
Comment (ChenPeleg) [inline src/contrast/pages/UploadNewPaper/ExperimentsSection/ExperimentForm.jsx:145]: This is not good. A hook is not supposed to be inside a conditional:

see [this](https://legacy.reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level)

File: src/contrast/pages/UploadNewPaper/ExperimentsSection/Findings.jsx [hunk 3]
```diff
@@ -148,7 +147,15 @@ export default function Findings({
                           setFieldValues(newArray);
                         }}
                         options={alphabetizeByLabels(
-                          fieldOptions.techniquesOptions
+                          // triming the list of option so no duplicated options
+                          fieldOptions.techniquesOptions.filter(
+                            (obj, index, self) =>
```
Comment (ChenPeleg) [inline src/contrast/pages/UploadNewPaper/ExperimentsSection/Findings.jsx:152]: What is this supposed to do? It's not clear

**Review by ChenPeleg — CHANGES_REQUESTED** [review]
Please Change the conditional hook (must) and (preferred) the unclear filter

File: src/contrast/pages/UploadNewPaper/ExperimentsSection/Findings.jsx (hunk 3 above)
Comment (oryrosin) [reply src/contrast/pages/UploadNewPaper/ExperimentsSection/Findings.jsx:152]: its written in comment on the row above

File: src/contrast/pages/UploadNewPaper/ExperimentsSection/ExperimentForm.jsx (hunk 2 above)
Comment (ChenPeleg) [reply src/contrast/pages/UploadNewPaper/ExperimentsSection/ExperimentForm.jsx]: This is not fixed yet

**Review by ChenPeleg — APPROVED** [review]

**Pull request merged**